subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

```diff
@@ -287,10 +301,10 @@ Since Checkstyle runs as an external process you can control the memory via the
 
 ==== Missing files specified with relative paths when running Checkstyle
 
-Gradle 7.5 consistently sets the current working directory for the Checkstyle task to `GRADLE_USER_HOME/workers`. 
+Gradle 7.5 consistently sets the current working directory for the Checkstyle task to `GRADLE_USER_HOME/workers`.
 This may cause problems with custom Checkstyle tasks or Checkstyle configuration files that assume a different directory for relative paths.
 
-Previously, the current working directory would depend on how Gradle was executed. When running Gradle from the root directory of a project, this would be the root directory, but when running Gradle from a subproject, the current working directory would be the subproject directory. 
+Previously, the current working directory would depend on how Gradle was executed. When running Gradle from the root directory of a project, this would be the root directory, but when running Gradle from a subproject, the current working directory would be the subproject directory.
```

Comment:
@nathan-contino requested some changes when I did the same trailing space changes on `master`: https://github.com/gradle/gradle/pull/19158/commits/b9a39be49183fae900bd2b330bde40d5a7cbb8c1

Can you cherry-pick that commit into your branch, so we already have the improvements for 7.6 and no conflicts between `master` and `release`?

Reply:
Done